src/opendap/build_dmrpp/BuildDmrppDispatchHandler.java

```diff
@@ -147,7 +149,7 @@ public boolean requestDispatch(HttpServletRequest request,
             boolean itsJustThePrefixWithoutTheSlash = false;
             boolean itsJustThePrefix = false;
 
-            boolean itsJustTheSlash = resourceID.equals(THE_SLASH) && _prefix.equals(THE_SLASH);
+            boolean itsJustTheSlash = resourceID.equals(THE_SLASH);//&& _prefix.equals(THE_SLASH);
```

Comment:
THis is the change that fixed the landing page.

Reply:
The commented code should be removed. If we need to keep it, then we should say so.

Reply:
I can do this if you're off. And then run the builds and deployment.